script/lib/util.py

```diff
@@ -246,3 +246,15 @@ def get_electron_exec():
 
   raise Exception(
       "get_electron_exec: unexpected platform '{0}'".format(sys.platform))
+
+def get_path_in_buildtools(name):
```

Comment:
```suggestion
def get_buildtools_executable(name):
```

since it has `.exe` logic